tests/DompdfTest.php

```diff
@@ -49,6 +49,9 @@ public function testSetters()
         $this->assertInstanceOf(Options::class, $dompdf->getOptions());
         $this->assertEquals('test3', $dompdf->getProtocol());
         $this->assertInstanceOf(FrameTree::class, $dompdf->getTree());
+
+        $dompdf->setHttpContext(['ssl' => ['verify_peer'=>false]]);
+        $this->assertIsResource($dompdf->getHttpContext());
```

Comment:
Would be nice to add tests for the new option to `OptionsTest`, too.

Reply:
Added